src/mmw/apps/modeling/serializers.py

```diff
@@ -70,14 +70,34 @@ class Meta:
         fields = ('id', 'name', 'project', 'is_current_conditions', 'inputs',
                   'inputmod_hash', 'modifications', 'modification_hash',
                   'aoi_census', 'modification_censuses', 'results',
-                  'created_at', 'modified_at')
+                  'created_at', 'modified_at', 'weather_type',
+                  'weather_simulation', 'weather_custom')
+        read_only_fields = ('weather_custom',)
```

Comment:
`weather_custom`, which is just the path to the file, can be read by the client (e.g. for checking if custom weather is available on a scenario), but cannot be written to. It is a `FileField`, and requires special writing code, which is in the `custom_weather_data` view.